systems/subscriber/sim-manager/pkg/clients/providers/notification.go

```diff
@@ -0,0 +1,93 @@
+package providers
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/url"
+
+	"github.com/go-resty/resty/v2"
+	"github.com/sirupsen/logrus"
+	"github.com/ukama/ukama/systems/common/rest"
+)
+
+type SendEmailReq struct {
+	To      []string          `json:"to" validate:"required"`
+	Subject string            `json:"subject" validate:"required"`
+	Body    string            `json:"body" validate:"required"`
+	Values  map[string]string `json:"values"`
+}
+
+type NotificationClient interface {
+	SendEmail(body SendEmailReq) error
+}
+
+type notificationClient struct {
+	RestClient *RestClient
+}
+
+type Notification struct {
```

Comment:
What is this used for ?

Reply:
The line type Notification struct { Client NotificationClient json:"notification" } defines a struct type named Notification with a field named Client of type NotificationClient. This struct is used to represent a notification object, where Client is responsible for sending the notification.

Reply:
I was asking for the `Notification` , It seems you are not using it anywhere.